modules/rosa-sts-about-ocm-role.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// rosa_planning/rosa-sts-ocm-role.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-about-ocm-role_{context}"]
+= About the ocm-role IAM role
+
+You need to create the `ocm-role` IAM resource to enable a Red Hat organization of users to create ROSA clusters.
+
+Some considerations for your `ocm-role` IAM resource are:
+
+* You have only one `ocm-role` per Red Hat organization.
+* Any user in a Red Hat organization may create and link an `ocm-role` IAM resource.
+* Only the Red Hat organization administrator can unlink an `ocm-role` IAM resource. This limitation is to protect other Red Hat organization members from disturbing the interface capabilities of other users.
++
+[NOTE]
+====
+If you just created a Red Hat account that is not part of an existing organization, this account is also the organization admin.
```

Comment:
@dayleparker can you explain some more? I don't see those details when I click that link. Unfortunately, I also don't see it on https://console.redhat.com either. 

@arendej is there a CLI command I should add in addition to the OCM info?

Reply:
@EricPonvelle When you are logged into console.redhat.com, your username shows on the top right corner of the screen - if you click your username and a dropdown shows you more info about your account. If you're an Organization Administrator, you can verify it there.

It's not essential to mention it, but I thought it could be helpful - totally up to you. :)
However, can you please edit 'the organization admin' to "the Organization Administrator"?

Reply:
> is there a CLI command I should add in addition to the OCM info?
I don't follow